devices/tuya.js

```diff
@@ -626,6 +626,45 @@ module.exports = [
             }
         },
     },
+    {
+        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3000_cphmq0q7'},],
+        model: 'TS011F',
+        description: 'Smart EU Plug',
+        vendor: 'TuYa',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report, fz.tuya_switch_power_outage_memory],
+        toZigbee: [tz.on_off, tz.tuya_switch_power_outage_memory],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'haElectricalMeasurement']);
+            endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 100, multiplier: 1});
+            endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {
+                acVoltageMultiplier: 1, acVoltageDivisor: 1, acCurrentMultiplier: 1, acCurrentDivisor: 1000, acPowerMultiplier: 1,
+                acPowerDivisor: 1,
+            });
+        },
+        // This device doesn't support reporting correctly.
+        // https://github.com/Koenkk/zigbee-herdsman-converters/pull/1270
```

Comment:
Are you sure this device does not support reporting? Can you check if power reporting also works when:
- Removing the `onEvent`
- Replacing configure with:

```js
configure: async (device, coordinatorEndpoint, logger) => {
    const endpoint = device.getEndpoint(1);
    await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'haElectricalMeasurement']);
    await reporting.rmsVoltage(endpoint);
    await reporting.rmsCurrent(endpoint);
    await reporting.activePower(endpoint);
    await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);
},
```

Reply:
Thank you! It does work like that. I've pushed the changes

Reply:
I was wrong. It doesn't update. I'll revert and create another PR